File: VB/Tiktok Login Design/UserControl/UserControlSign Up.vb
Public Class UserControlSign_Up
    Private Sub LinkLabel3_LinkClicked(sender As Object, e As LinkLabelLinkClickedEventArgs) Handles LinkLabel3.LinkClicked
        Form1.Guna2Panel1.Controls.Clear()
        Form1.Guna2Panel1.Controls.Add(New UserControlLogin())
    End Sub

    Private Sub Guna2GradientButton2_Click(sender As Object, e As EventArgs) Handles Guna2GradientButton2.Click
        Form1.Guna2Panel1.Controls.Clear()
        Form1.Guna2Panel1.Controls.Add(New UserControlSignUpCreatAcount())
    End Sub
End Class

File: C#/Tiktok Login Design/UserControl/UserControlSign Up.cs
using System;
using System.Windows.Forms;

namespace Tiktok_Login_Design
{
    public partial class UserControlSign_Up
    {
        public UserControlSign_Up()
        {
            InitializeComponent();
        }
        private void LinkLabel3_LinkClicked(object sender, LinkLabelLinkClickedEventArgs e)
        {
            My.MyProject.Forms.Form1.Guna2Panel1.Controls.Clear();
            My.MyProject.Forms.Form1.Guna2Panel1.Controls.Add(new UserControlLogin());
        }

        private void Guna2GradientButton2_Click(object sender, EventArgs e)
        {
            My.MyProject.Forms.Form1.Guna2Panel1.Controls.Clear();
            My.MyProject.Forms.Form1.Guna2Panel1.Controls.Add(new UserControlSignUpCreatAcount());
        }
    }
}